src/ingestors/transient-base/offchain-metadata.ts

```diff
@@ -109,7 +109,7 @@ export const getTransientBaseMintBySlug = async (
   const { nft_contract, media_url, name } = data;
 
   return {
-    chainId: nft_contract.address.chain,
+    chainId: parseInt(nft_contract.address.chain),
```

Comment:
This was the bug I went after - caused ecomm not to be able to save the mint instructions